guides/tutorials/authenticate_with_service_account.md

```diff
@@ -18,14 +20,14 @@ Before starting, ensure you have:
 Open your terminal and run the following command:
 
 ```
-atlas auth login
+atlas auth login --profile myProfile
 ```
 
-This command initiates the authentication setup for your Atlas CLI profile.
+This command initiates the interactive authentication setup guide for a Atlas CLI profile 'myProfile'.
```

Comment:
Super minor but felt like a word was missing here: "...setup guide for a Atlas CLI profile **_called_** 'myProfile'"